packages/camera/lib/camera.dart

```diff
@@ -49,12 +49,9 @@ CameraLensDirection _parseCameraLensDirection(String string) {
 /// May throw a [CameraException].
 Future<List<CameraDescription>> availableCameras() async {
   try {
-    final List<dynamic> cameras =
-        // TODO(amirh): remove this on when the invokeMethod update makes it to stable Flutter.
-        // https://github.com/flutter/flutter/issues/26431
-        // ignore: strong_mode_implicit_dynamic_method
-        await _channel.invokeMethod('availableCameras');
-    return cameras.map((dynamic camera) {
+    final List<Map<String, dynamic>> cameras = await _channel
```

Comment:
Since the result is a **list** of maps, you would need to replace `String` with `dynamic`. Unfortunately, maps don't have implicit casting and this would throw the `_LinkedHashMap<dynamic, dynamic>` error.

```
final List<Map<dynamic, dynamic>> cameras = await _channel
    .invokeListMethod<Map<dynamic, dynamic>>('availableCameras');
```

Reply:
@bparrishMines done, could you take a look again just to make sure?

Reply:
Pulled your changes in and everything still works for me!

Reply:
cool! I will merge once the builds are green!